src/main/java/org/owasp/html/HtmlStreamRenderer.java

```diff
@@ -252,23 +253,20 @@ private final void writeCloseTag(String uncanonElementName)
         StringBuilder cdataContent = pendingUnescaped;
         pendingUnescaped = null;
         Encoding.stripBannedCodeunits(cdataContent);
-        int problemIndex = checkHtmlCdataCloseable(lastTagOpened, cdataContent);
+        problemIndex = checkHtmlCdataCloseable(lastTagOpened, cdataContent);
         if (problemIndex == -1) {
           if (cdataContent.length() != 0) {
             output.append(cdataContent);
           }
         } else {
-          error(
-              "Invalid CDATA text content",
-              cdataContent.subSequence(
-                  problemIndex,
-                  Math.min(problemIndex + 10, cdataContent.length())));
-          // Still output the close tag.
+            //don't bother throwing exception for not getting closed tags , rather we skip rendering it in HTML
```

Comment:
instead of just skipping it entirely, can we at least put a log statement at warning level?

Reply:
So I tried adding logger two ways 
1) Directly add the Sysout statement - Somewhere in code, it's mentioned if you don't want an error to be thrown, replace with `sysout.err`
2) using Log4z to add loggers. for this I had to add dependency and create a static variable and log the statement in warning level

Let me know which I should go ahead with
Both the logs are available in `zmmailboxd.out` file
OWASP code doesn't have a predefined way of adding a logger hence had to add dependency and do logging stuff

Reply:
@zimsuchitgupta @RakeshAMore Can you please check and suggest best approach here?

Reply:
@gabhinavTI this is really simple :) add log4j warning statement.